issue_template.md

```diff
@@ -19,6 +19,10 @@ Checkout how to troubleshoot failures and collect debug logs: https://docs.micro
 
 [Include task name(s), screenshots and any other relevant details]
 
+### Task logs
+
+[Enable debug logging and please provide the zip file containing all the logs for a speedy resolution]
```

Comment:
[Enable debug logging and please provide the zip file containing all the logs] enough 🍔